xbmc/cores/VideoPlayer/DVDSubtitles/webvtt/WebVTTHandler.cpp

```diff
@@ -273,20 +276,19 @@ void CWebVTTHandler::DecodeLine(std::string line, std::vector<subtitleData>* sub
             locHrs = std::stoi(regLocal.GetMatch(2).c_str());
           locMins = std::stoi(regLocal.GetMatch(3).c_str());
           locSecs = std::atof(regLocal.GetMatch(4).c_str());
-          m_hlsTimestampLocalUs =
-              (static_cast<double>(locHrs * 3600 + locMins * 60) + locSecs) * DVD_TIME_BASE;
+          tsLocalUs = (static_cast<double>(locHrs * 3600 + locMins * 60) + locSecs) * DVD_TIME_BASE;
           // Converts a 90 kHz clock timestamp to a timestamp in microseconds
-          m_hlsTimestampMpegTsUs =
-              std::stod(regMpegTs.GetMatch(1)) * MICROS_PER_SECOND / MPEG_TIMESCALE;
+          tsMpegUs = std::stod(regMpegTs.GetMatch(1)) * MICROS_PER_SECOND / MPEG_TIMESCALE;
```

Comment:
`std::stod` can return an exception. It would be much safer to handle this properly as you are modifying the code.

Reply:
nope all use of stod in this file are already safe by regex checks

Reply:
Ok, cool.